src/main/java/uk/gov/hmcts/reform/juddata/camel/exception/RouteFailedException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.juddata.camel.exception;
+
+public class RouteFailedException extends RuntimeException {
+
+    public RouteFailedException(String message) {
+        super(message);
+    }
+}
```

Comment:
is not there a test class for this?

Reply:
gets covered from integration/unit cases.Mocking won't add much value.